pkg/transcriptionsystems/bipa/consonants.tsv

```diff
@@ -606,3 +609,10 @@ kʲ	voiceless	velar	stop		palatalization:palatalized
 k̚	voiceless	velar	stop		release:unreleased	
 kʷʼ	voiceless	velar	stop	+	labialization:labialized,ejection:ejective	
 ɡ̤	voiced	velar	stop	+	breathiness:breathy	
+d̪l̪	voiced	lateral	affricate			
+ɽr	voiced	retroflex	trill			
+kp	voiceless	labio-velar	stop			
+gb	voiced	labio-velar	stop			
+qʡ	voiceless	uvular-epiglottal	stop			
+nm	voiced	labio-alveolar	nasal			
```

Comment:
please, we do not need new features like labio-alveolar. You can as well just handle them as double-articulation.

Reply:
I have only added "labio-alveolar" and "uvular-epiglottal", the other double-articulated places are already in use. labio-palatalization can also be given as a value of the "palatalization" feature, which can be somewhat redundant (IPA gives ɥ as a labialized palatal, but this is to a point dependent on the framework of analysis).

Reply:
As this is only the `nm` case, which is a cluster, there's no need to add it then.